docs/overview/index.md

```diff
@@ -22,7 +22,7 @@ The services have maximum numbers of queries you can make within a certain perio
 
 All the projects used to build the Mapzen-hosted services are open source. If you want to try Mapzen's products, start with the hosted services to see if they fit your workflow needs. If you later decide that you need additional customizations or higher capacity, you can consider installing on your own servers the open-source code used to build Mapzen's services.
 
-If you send a query without a valid API key (keyless access), the rate limits for Mapzen Search, Turn-by-Turn, Matrix, and Elevation are reduced to 1,000 requests per day, 6 per minute, and 1 per second.
+If you send a query without a valid API key (keyless access), the rate limits for Mapzen Search, Turn-by-Turn, Matrix, Elevation, and Vector Tiles are reduced to 1,000 requests per day, 6 per minute, and 1 per second.
```

Comment:
remove the service names here if they are all the same now? "If you send a query without a valid API key (keyless access), the rate limits are 1,000 requests per day, 6 per minute, and 1 per second for each Mapzen service."